react/features/video-quality/components/VideoQualityLabel.web.js

```diff
@@ -143,21 +164,21 @@ export class VideoQualityLabel extends Component {
         const classNames
             = `${baseClasses} ${filmstrip} ${opening}`;
 
-        let labelContent;
+        let labelIcon;
         let tooltipKey;
 
         if (_audioOnly) {
-            labelContent = <i className = 'icon-visibility-off' />;
+            labelIcon = AUDIO_ONLY_ICON;
             tooltipKey = 'videoStatus.labelTooltipAudioOnly';
         } else if (!_videoTrack || _videoTrack.muted) {
-            labelContent = <i className = 'icon-visibility-off' />;
+            labelIcon = AUDIO_ONLY_ICON;
             tooltipKey = 'videoStatus.labelTooiltipNoVideo';
         } else {
-            const translationKeys
-                = this._mapResolutionToTranslationsKeys(_resolution);
+            const displayConfig
+                = this._mapResolutionToDisplayConfig(_resolution);
```

Comment:
You shouldn't merge anyway until I change the icons to be done in HTML instead of font icons. I added you as a reviewing because that change should directly affect your work in the near future.